packages/expo-modules-core/ios/JSI/EXJavaScriptRuntime.mm

```diff
@@ -198,6 +198,23 @@ - (nonnull EXJavaScriptObject *)createSharedObjectClass:(nonnull NSString *)name
   return [[EXJavaScriptObject alloc] initWith:klass runtime:self];
 }
 
+#pragma mark - Shared refs
+
+- (nonnull EXJavaScriptObject *)createSharedRefClass:(nonnull NSString *)name
```

Comment:
I wonder if we can easily merge it with `createSharedObjectClass`, which is almost the same 🤔